docs/migration/test-cloud/index.md

```diff
@@ -1,13 +1,94 @@
+
+In order to start uploading to App Center, you must install the App Center CLI. To install it, please use:
+
+```
+npm install -g appcenter-cli
+```
+Read more about the [appcenter-cli]((https://github.com/Microsoft/appcenter-cli) and how to upload to App Center.
+
+**Q:** Is the team that built App Center Test different than Xamarin Test Cloud? 
+
+**A:** Visual Studio App Center Test is built and maintained by the same team as Xamarin Test Cloud, so we’re confident you’ll love what it has to offer. If you have any concerns at all, please contact your account manager or reach out to us at appcentertest@microsoft.com.
+
+**Q:** *How is support handled in App Center?*
+
+**A:** Support will now be handled in [Intercom](https://intercom.help/appcenter). If you have a question you can click the blue circle at the bottom right-hand side of your App Center dashboard to begin a conversation with one of agents.
+
+**Q:** *What do I need to do during this upgrade?*
+
+**A:** We'll handle the upgrade for you. On April 1st, you will simply log into App Center using your Xamarin Test Cloud account.
+
+**Q:** *Can I upgrade to App Center before March 31st?*
+
+**A:** You don’t have to wait until March 31st. App Center Test is fully functional and fully supported. If you have not already done so, please create a Visual Studio App Center account using the same email address as your Xamarin Test Cloud account. You can then familiarize yourself with all the features of Visual Studio App Center with the knowledge that we will migrate your Xamarin Test Cloud data into your new account on March 31st.
+
+**Q:** *How is support handled in App Center?*
+
+**A:** Support will be handled in [Intercom](https://intercom.help/appcenter). If you have a support question you can click the blue circle at the bottom right-hand side in App Center to initiate a conversation with one of our agents.
+
+**Q:** What do I have to do if I have an App Center account already?
+
+**A:** On March 31, 2018, as part of the upgrade, we will import your teams, orgs, users and test data into the Visual Studio App Center account with the same email address as your Xamarin Test Cloud account. If you do not have a Visual Studio App Center account at that time, we'll create one using an email address identical to the email you use in Xamarin Test Cloud. If you currently have an App Center account, the email of the account must match your Xamarin Test Cloud email address exactly for this import to occur.
+
+**Q:** *Will my service be interrupted?*
+
+**A:** All Xamarin Test Cloud accounts will remain active until March 31, 2018. Your current tests will continue to work through that date. You will be able to create and run new tests as usual during that time and there will be no disruption of your current service. After March 31, 2018, the Xamarin Test Cloud portal will redirect to the Visual Studio App Center portal, and all testing will be performed through there moving forwards.
```

Comment:
"will be performed through there" could be tightened up a bit. How about something like "After March 31, 2018, the Xamarin Test Cloud portal will redirect to the Visual Studio App Center portal, and all future testing happens in App Center."